### PR TITLE
Please be kind 👏🏻

### PR DESCRIPTION
**Pull request by iamdanielcooper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
...
+
+## Installation & Usage
+
+- Clone / download this repo.
+- In your terminal navigate to `cd thoughts`
+- To start enter `` into your terminal.
+- Open the client `./client/index.html`
+- The api will now be serving at `https://localhost:3000`
+- To close enter `` into your terminal.
+
```
Comment (humza1997) [inline README.md:27]: Nice installation guide!

File: README.md [hunk 2]
```diff
@@ -1 +1,63 @@
...
+Our data is stored in in a relational database with the below schema.
+
+| Name      | Type    | notes              |
+| --------- | ------- | ------------------ |
+| ID        | Serial  | Primary Key        |
+| Title     | Varchar | Max 100 characters |
+| Pseudonym | Varchar | Max 100 characters |
+| body      | Varchar | Max 500 characters |
+
```
Comment (humza1997) [inline README.md:55]: I like this table :)

File: README.md [hunk 3]
```diff
@@ -1 +1,63 @@
...
+## Routes
+
+Our app uses the below restful routes
+
+| Method | Endpoint   | Info                             |
+| ------ | ---------- | -------------------------------- |
+| POST   | /posts/new | Adds a new entry to the database |
+| GET    | /posts/:id | gets a post by id                |
```
Comment (Samekah) [inline README.md:63]: I really like your readme, it's layed out quite nicely. its easy to read and has all the necessary information!

File: api/dbConfig/dev_seeds.sql [hunk 4]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO thoughts (title, pseudonym, body) 
+VALUES
+('Test seed 1', 'Edward Elric', "Damn it, there are so many idiots whose asses I have to kick! I'll have to start carrying a list just to keep track of 'em all!"),
+('Test seed 2', 'Monkey D Luffy', "I Decided To Be Pirate King. I Don’t Care If I Die For It."),
```
Comment (Samekah) [inline api/dbConfig/dev_seeds.sql:4]: same, big mood XD

File: api/dbConfig/dev_seeds.sql [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+('Test seed 3', 'Naruto Uzumaki', "ichiraku ramen!!!");
```
Comment (Samekah) [inline api/dbConfig/dev_seeds.sql:5]: nice seeding!

File: api/models/thoughts.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+                console.log('posted to server')
+                resolve(thoughtPost);
+            } catch (err) {
+                reject('Something went wrong, could not create though post :-(')
+            }
+        });
+    }
+};
+
+module.exports = Thoughts;
```
Comment (Samekah) [inline api/models/thoughts.js:37]: Nice error handling and prevention of SQL injections.
Really good SQL statements.

File: client/static/css/js/index.js [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(output)
+    }
+    
+
+    // TODO This should return the id of the newly recrated object.
+    const response = await fetch("http://localhost:3000/posts/new", options);
+
```
Comment (humza1997) [inline client/static/css/js/index.js:30]: We were attempting something similar, In theory if you can access the id from the object array thats produced on input, you could append it onto a string that redirects to the next html page. And then on the next html page, you can access that fetched data using a show path from your route.

File: db/1_create_thoughs.sql [hunk 8]
```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS thoughts;
+
+CREATE TABLE thoughts(
+    id serial,
```
Comment (Samekah) [inline db/1_create_thoughs.sql:4]: Possibly add a `PRIMARY KEY` so that there is no duplication of  data

File: README.md [hunk 9]
```diff
@@ -1 +1,63 @@
...
+  - [Routes](#routes)
+
+## Notes
+- The route to view the past posts isn't finished yet.
+
+## Installation & Usage
+
+- Clone / download this repo.
+- In your terminal navigate to `cd thoughts`
+- To start enter `` into your terminal.
```
Comment (Samekah) [inline README.md:23]: Enter what?

**Review by Samekah — COMMENTED** [review]
Really love the animation when you make a post. Good RESTFUL routing.

**Comment by iamdanielcooper** [comment]
Thanks guys :)

**Pull request closed without merging**